src/Users.php

```diff
@@ -54,6 +55,10 @@ public function saveUser($user)
             $user = new Entity\Users($user);
         }
 
+        $hasher = new PasswordLib();
+
+        $user->setPassword($hasher->createPasswordHash($user->getPassword()));
+
```

Comment:
This one still needs to go, or you will end up with hashed password hashes, that have hashes in their hash-hash… it would be like Hashception :grin: